server/router.ts

```diff
@@ -2,32 +2,55 @@ import { Router, Request, Response, NextFunction } from 'express'
 import axios from 'axios'
 import {
   accessUserUrl,
-  githubCallbackUrl,
-  githubLoginUrl,
   githubTokenUrl
 } from './config/github-config'
+import { PrismaClient } from '@prisma/client'
+import {encodeJwt} from './utils/jwt'
+import queryParser from 'query-parser-url'
 
+const prisma = new PrismaClient()
```

Comment:
지금 graphql 폴더에서도 new PrismaClient가 실행되어서, 프리즈마 클라이언트 객체가 2개가 생성되는 상태예요
그래서 나중에 prisma 폴더안에 index.ts 만들고 싱글톤 형식으로 export 해주는 걸 고려해야 할 것 같아요!
이건 백로그에 작성을 해놓을게요!

```suggestion
// prisma/index.ts
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()
export default prisma
```

Reply:
네 알겠습니다~!